pyxcli/transports.py

```diff
@@ -154,7 +154,7 @@ def __repr__(self):
                                                    h, p, ssl)
 
     @classmethod
-    def connect(cls, hostname, port, timeout=5.0):
```

Comment:
Did the network get slower these days? You need a **very** good reason to change this.

Reply:
@alonm, for HyperSwap/multisite creation and deletion scenario, commands take more than 5 seconds to respond.  sometimes it takes 30s.

Can we change timeout to a configurable value, so third party module can change this value on demands ?

Reply:
This is an optional parameter. The third-party module can specify this value on demand - so there is no reason to change this.

Reply:
@alonm, XIV cinder driver use this method to initialize xcli connection
`        try:
            LOG.debug('connect_multiendpoint_ssl with: %s', address)
            xcli = client.XCLIClient.connect_multiendpoint_ssl(
                user,
                clear_pass,
                address,
                ca_certs=path)`

but this method does not support timeout parameter
`    def connect_multiendpoint_ssl(cls, user, password, endpoints,
                                  auto_discover=True, ca_certs=None,
                                  validate=None):`